torch/autograd/forward_ad.py

```diff
@@ -4,6 +4,8 @@
 
 from typing import Any
 
+__all__ = ["enter_dual_level", "exit_dual_level", "make_dual", "unpack_dual", "dual_level"]
```

Comment:
doc please

Reply:
Could you add the following line?
```
UnpackedDualTensor.__doc__ = r"""Namedtuple returned by :func:`unpack_dual` containing the primal and tangent components of the dual tensor."""\
"""See :func:`unpack_dual` for more details. ""
```

Reply:
It would be nice to avoid `__doc__` assignment
Maybe doing 
```python
class UnpackedDualTensor(NamedTuple):
    """The doc here"""
    primal: Tensor
    tangent: Tensor
```

Reply:
Apparently there is also:
```
class Point(namedtuple('Point', 'x y')):
    """Here is the docstring."""
```
but we would need to specify `__slots__ = ()` or else a dict would be created for the class.
See: https://stackoverflow.com/questions/1606436/adding-docstrings-to-namedtuples

Reply:
added the doc!